app/actions/v2/app_update.rb

```diff
@@ -173,6 +177,10 @@ def is_web_process_deploying(mass_assign, process)
       def staging_necessary?(process, request_attrs)
         request_attrs.key?('state') && process.needs_staging?
       end
+
+      def v2_api_staging_disabled?
+        !!VCAP::CloudController::Config.config.get(:temporary_disable_v2_staging)
```

Comment:
Yes, I want to coerce nil into false since this is an optional config property. In the schema it is set to be a `bool`, so I don't expect there to be any strings.

Reply:
Regarding your comment above:

> is it possible to make the api_schema field optional & bool with a default?

We traditionally did this by making it have a default value in `capi-release`, we currently don't have a generic solution for defaulting config right now in CCNG as far as I know.